.circleci/config.yml

```diff
@@ -422,8 +422,8 @@ jobs:
     <<: *contrib_job
     parallelism: 8
```

Comment:
can we drop the parallelism now? probably don't need it if running with riot

Reply:
do we even have 8 Python runtimes for this job?